### ISSUE TITLE
setting up project

### ISSUE THREAD

**Issue by Qweder93** [gitbitex/gitbitex-new#21]
Hello, I made all steps from README
After docker ps I can see 
CONTAINER ID   IMAGE                      COMMAND                  CREATED          STATUS          PORTS     NAMES
0c10591a37ba   greensheng/gitbitex        "java -XX:ErrorFile=…"   45 seconds ago   Up 44 seconds             gitbitex
93b3fce13d89   mysql:5.7                  "docker-entrypoint.s…"   16 minutes ago   Up 16 minutes             mysql
5663e375b58f   redis                      "docker-entrypoint.s…"   16 minutes ago   Up 16 minutes             redis-server
aa1feba79f7d   bitnami/kafka:latest       "/opt/bitnami/script…"   17 minutes ago   Up 17 minutes             kafka-server
6f6ab3a5f2ee   bitnami/zookeeper:latest   "/opt/bitnami/script…"   18 minutes ago   Up 18 minutes             zookeeper-server

I have everything running, still nothing on https://127.0.0.1/trade/BTC-USDT, looks like app doesn't run. Any suggestions? Thanks a lot!

Also another small question: I found you had a Golang project repo which is abandoned. Is it working? Is there some issues with it or any reasons it should not be used? Thanks!

**Comment by greensheng** [comment]
What operating system are you using? It may be that the -- network=host parameter does not work properly

**Comment by Qweder93** [comment]
macOS

**Comment by Qweder93** [comment]
also if you can tell a few words what status of Golang project (I know its abandoned, but how complete is it?)

**Comment by greensheng** [comment]
> also if you can tell a few words what status of Golang project (I know its abandoned, but how complete is it?)

I don't have much Golang experience. That project is just for learning Golang.

**Comment by opadfnezig** [comment]
I've managed to run it on Windows, but having problems with websocket i guess. Here is compose image I've created. 
[docker-compose.zip](https://github.com/gitbitex/gitbitex-new/files/10851533/docker-compose.zip)
You'll have to reconfigure project a bit, in src/main/resources/application.properties

spring.datasource.url=jdbc:mysql://mysql:3306/gitbitex
kafka.bootstrap-servers=kafka-server:9092
redis.address=redis://redis-server:6379

Then 
`mvn clean compile jib:dockerBuild`
and run compose.
Also, don't remember why, but I had to remove tests. You might also have to do it.
